doc/ref/run.xml

```diff
@@ -391,9 +399,10 @@ redirect the output to a printer, you may want to increase this value.
 <Item>
 Further arguments are taken as filenames of files that are read by &GAP;
 during startup, after the system  and private init   files are read,  but
-before the first prompt is  printed.
-The files are read in the order in which they appear on the command line.
-&GAP; only accepts up to 14 filenames on the command line.
+before the first prompt is  printed. Filenames ending with <C>.tst</C>
+are processed by <Ref Func="Test" />, all other files by <Ref Oper="Read" />.
+These files and also commands specified via the <C>-c</C> option are processed
+in the order in which they appear on the command line.
 If a file cannot be opened &GAP; will print an error message and will abort.
```

Comment:
Since this last sentence is wrong, it should be corrected in this pull request.

I would prefer that the behaviour is not changed and only the documentation is adjusted. Currently, one can do some debugging in the running GAP.

How about:
```
If a file cannot be opened by &GAP; or an argument of  the <C>-c</C> option cannot be evaluated then &GAP; will print an error message and continue to start up.
```

Reply:
Of course the situation is even more complex, because one also should explain what happens with additional filenames resp. `-c` options after the one which failed: if the "failure" was a syntax error, we continue processing the others; if it was an error then we abort processing the others. That's IMHO not very intuitive, so it should then also be explained.

Reply:
I've tried to write something and thought some more, and I really don't see how this final behavior is sensible or useful: if at all, shouldn't it be reverse? I.e. if the user decides to proceed after an `Error`, after being prompted what to do, we may as well continue to process, as the user is well-aware there was a problem. On the other hand, a syntax error is already just a message; it can easily be overlooked esp. if subsequent code prints a lot. So by effectively ignoring it, we make debugging *harder* in my eyes; I'd thus understand if we aborted processing after a syntax error.

On the other hand, if the idea really is that syntax errors are "fine" (why, though??), then why do we still abort processing of the file the syntax error occurred in (by using `READ_NOCOVERY` instead of `READ`) yet happily proceed to process the next file?

This really don't make sense to me....


That all said: I'd rather merge this PR first, which contains a genuine improvement, and *then* worry about the broken documentation text (which was around for years and years and nobody noticed), instead of holding the PR hostage...